### PR TITLE
Fvm 031 new work page

### PR DESCRIPTION
**Pull request by stevenrayesky**

### CODE REVIEW

**Comment by jpickel** [comment]
@stevenrayesky There are two conflicts on this branch (resulting from the contact page that was merged with master that we launched last week). Can you resolve those conflicts on this work branch first before I review?

**Comment by stevenrayesky** [comment]
Ok resolved the conflicts.

File: wp-content/themes/fvm/loop-contact.php [hunk 1]
```diff
@@ -52,8 +52,13 @@
 
     <?php include (TEMPLATEPATH . '/include-social.php' ); ?>
     
+<<<<<<< FVM-031-New-Work-Page
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-contact.php]: There's merge conflict code in here

File: wp-content/themes/fvm/loop-work.php [hunk 2]
```diff
@@ -1,17 +1,16 @@
-<?php 
-	$this_page_id=$wp_query->post->ID;
-	$id = get_the_ID();
-?>
-<?php if (have_posts()) : while (have_posts()) : the_post(); ?>
-
     <section id="intro" class="page block r4x3">
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-work.php]: The left side grey block is missing on a category view: http://localhost:8888/thinkfvm/tags/positioning/

File: wp-content/themes/fvm/loop-work.php [hunk 3]
```diff
@@ -37,23 +79,70 @@
...
+	    <?php } ?>
 
-	<?php else : ?>
+		<div class="block r2x2">
+			<div class="outer r2x2">
+				<div class="inner services-cta bg black">
+					<div class="paginate col-md-12 bg dark-grey">
+						<div class="paginate__next"><?php next_posts_link( 'Next', $query->max_num_pages ); ?></div>
+					</div>
+					<div class="paginate col-md-12 bg medium-grey">
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-work.php]: Is next and previous supposed to have two different background colors?

File: wp-content/themes/fvm/loop-work.php [hunk 4]
```diff
@@ -37,23 +79,70 @@
...
+				</div>
+			</div>
+		</div>
+		<div class="block r2x2">
+			<div class="outer r2x2">
+				<div class="inner bg orange flex contact-wrap">
+					<div class="contact-cta">
+						<h2 class="contact-cta__title">Ready to</h2>
+						<h2 class="contact-cta__title">get Started?</h2>
+						<h3 class="contact-cta__page"><a href="#" class="contact-cta__link">Contact Us</a></h3>
```
Comment (jpickel) [inline wp-content/themes/fvm/loop-work.php]: Link to contact page

File: wp-content/themes/fvm/loop-work.php (hunk 3 above)
Comment (stevenrayesky) [reply wp-content/themes/fvm/loop-work.php]: In the mockup it shows them as two separate colors. I made an adjustment with this -- you can't see a "Previous" unless you are a page other than the first (`previous_posts_link( 'Previous', $query->max_num_pages );` comes back as empty) so I made the box conditional so I that it only appears if there is something to show.

**Pull request merged**